tests/installable-build-test.ts

```diff
@@ -151,4 +151,29 @@ describe("installable build handling", () => {
 
       expectComment(webhook, `You can test the changes on this Pull Request by downloading the APK [here](${mockedArtifacts[0].url}).`)
     })
+
+    it("Posts a download comment linking to multiple APKs", async () => {
```

Comment:
Note that this unit test is just a shameless copy/paste of the previous unit test (lines 133-153) for single-APK case, adjusted to have a mockedArtifacts containing multiple APKs and a adjusted expectation.